gonative fails to build

```
$ godep go install
# github.com/inconshreveable/axiom
../axiom/axiom.go:108: too few values in struct initializer
../axiom/axiom.go:109: too few values in struct initializer
../axiom/axiom.go:110: too few values in struct initializer
../axiom/axiom.go:231: too few values in struct initializer
../axiom/axiom.go:330: too few values in struct initializer
godep: go exit status 2
$ go version
go version go1.5.1 darwin/amd64
$ godep version
godep v3 (darwin/amd64/go1.5)
```

Well, after creating a clean workspace:
```bash
$ go get -d github.com/inconshreveable/gonative
$ cd src/github.com/inconshreveable/gonative
$ make
go get github.com/tools/godep
WORKSPACE/src/github.com/inconshreveable/gonative/Godeps/_workspace/bin/godep restore
godep: restore, during download dep: exit status 1
make: *** [workspace] Error 1
```

--- a/axiom.go
+++ b/axiom.go
@@ -105,9 +105,9 @@ func NewLogged() *Logged {
 
 func (w *Logged) Wrap(cmd cli.Command) cli.Command {
 	cmd.Flags = append(cmd.Flags, []cli.Flag{
-		cli.StringFlag{"log", w.DefaultTarget, "path to log file, 'stdout', 'stderr' or 'false'", ""},
-		cli.StringFlag{"log-level", w.DefaultLevel, "logging level", ""},
-		cli.StringFlag{"log-format", w.DefaultFormat, "log record format: 'term', 'logfmt', 'json'", ""},
+		cli.StringFlag{"log", w.DefaultTarget, "path to log file, 'stdout', 'stderr' or 'false'", "", nil},
+		cli.StringFlag{"log-level", w.DefaultLevel, "logging level", "", nil},
+		cli.StringFlag{"log-format", w.DefaultFormat, "log record format: 'term', 'logfmt', 'json'", "", nil},
 	}...)
 	oldAction := cmd.Action
 	cmd.Action = func(c *cli.Context) {
@@ -228,7 +228,7 @@ func (w *YAMLConfigLoader) Load(rd io.Reader) error {
 
 func (w *YAMLConfigLoader) Wrap(cmd cli.Command) cli.Command {
 	cmd.Flags = append(cmd.Flags, []cli.Flag{
-		cli.StringFlag{"config", w.DefaultPath, "path to YAML config file", ""},
+		cli.StringFlag{"config", w.DefaultPath, "path to YAML config file", "", nil},
 	}...)
 	oldAction := cmd.Action
 	cmd.Action = func(c *cli.Context) {
@@ -327,7 +327,7 @@ func (cmd *Updater) Command() cli.Command {
 		Name:  "update",
 		Usage: "update to the latest version",
 		Flags: []cli.Flag{
-			cli.StringFlag{"channel", cmd.DefaultChannel, "update to the most recent release on this channel", ""},
+			cli.StringFlag{"channel", cmd.DefaultChannel, "update to the most recent release on this channel", "", nil},
 		},
 		Action: func(c *cli.Context) {
 			res, err := cmd.Update(c.String("channel"), c.App.Version)

